.editorconfig

```
@@ -1,7 +1,7 @@
 root = true
 [*.{cpp,h,py,txt,editorconfig}]
 
-indent_style = space
-indent_size = 2
+indent_style = tab
+tab_width = 4
 insert_final_newline = true
-trim_trailing_whitespace = true
\ No newline at end of file
+trim_trailing_whitespace = true

```

Let's keep the cmake and python scripts space indented.
(their indentation size can be changed in a later commit)

```suggestion
[*.{py,txt,editorconfig}]
indent_style = space
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = true

[*.{cpp,h}]
indent_style = tab
tab_width = 4
insert_final_newline = true
trim_trailing_whitespace = true
```